diff --git a/flake.lock b/flake.lock
index f13af1e..bf01939 100644
--- a/flake.lock
+++ b/flake.lock
@@ -113,11 +113,11 @@
         ]
       },
       "locked": {
-        "lastModified": 1753056897,
-        "narHash": "sha256-AVVMBFcuOXqIgmShvRv9TED3fkiZhQ0ZvlhsPoFfkNE=",
+        "lastModified": 1753848447,
+        "narHash": "sha256-fsld/crW9XRodFUG1GK8Lt0ERv6ARl9Wj3Xb0x96J4w=",
         "owner": "nix-community",
         "repo": "home-manager",
-        "rev": "13a83d1b6545b7f0e8f7689bad62e7a3b1d63771",
+        "rev": "d732b648e5a7e3b89439ee25895e4eb24b7e5452",
         "type": "github"
       },
       "original": {
@@ -128,11 +128,11 @@
     },
     "nixpkgs": {
       "locked": {
-        "lastModified": 1752950548,
-        "narHash": "sha256-NS6BLD0lxOrnCiEOcvQCDVPXafX1/ek1dfJHX1nUIzc=",
+        "lastModified": 1753694789,
+        "narHash": "sha256-cKgvtz6fKuK1Xr5LQW/zOUiAC0oSQoA9nOISB0pJZqM=",
         "owner": "nixos",
         "repo": "nixpkgs",
-        "rev": "c87b95e25065c028d31a94f06a62927d18763fdf",
+        "rev": "dc9637876d0dcc8c9e5e22986b857632effeb727",
         "type": "github"
       },
       "original": {
@@ -165,11 +165,11 @@
     "qtile-extras-flake": {
       "flake": false,
       "locked": {
-        "lastModified": 1753101774,
-        "narHash": "sha256-RHhw6O+lAJvU/xfZUEEc5cojrsJcTllCd+uoI6LTRGs=",
+        "lastModified": 1753640349,
+        "narHash": "sha256-vz5XiA1yDpstVRMoT8hPbGJjAEFCs06CLBPew0hUH7c=",
         "owner": "elparaguayo",
         "repo": "qtile-extras",
-        "rev": "0addf8d5d265737efcbcee68355573752e77aa88",
+        "rev": "82e28d57034181855d9883f015e5c446c6d88fe4",
         "type": "github"
       },
       "original": {
@@ -185,11 +185,11 @@
         ]
       },
       "locked": {
-        "lastModified": 1753101861,
-        "narHash": "sha256-alAJxc0PIpyigtyFiE7hSPEwQ6L9tpAO5yxkeeQIEI4=",
+        "lastModified": 1753579419,
+        "narHash": "sha256-hEYKWVGok8PsoUbXom3NskrSUa4y5kXgrl6SJ1ANUkY=",
         "owner": "qtile",
         "repo": "qtile",
-        "rev": "cbb896ed0b797caeab4fa65cc7f34ddbc74b1e8d",
+        "rev": "b667f9efd37eab117943f739b01e4d15bad1e09f",
         "type": "github"
       },
       "original": {
@@ -219,11 +219,11 @@
         "systems": "systems_2"
       },
       "locked": {
-        "lastModified": 1752986956,
-        "narHash": "sha256-1AK8+W7d5eNyGRkcWHa+9oIChLJbY6jt7ujSJo+ft4M=",
+        "lastModified": 1753591727,
+        "narHash": "sha256-Ow+qyFckroPS4SQFHcFZ8mKh3HIQ2pQdC6DRjiYF9EE=",
         "owner": "Gerg-L",
         "repo": "spicetify-nix",
-        "rev": "51ac0aee7e7ee21ca0874b913f07f9004bc9311f",
+        "rev": "26c488b60360e15db372483d826cec89ac532980",
         "type": "github"
       },
       "original": {
diff --git a/home-manager/modules/cli/tmux.nix b/home-manager/modules/cli/tmux.nix
index c6ab84b..89516f7 100644
--- a/home-manager/modules/cli/tmux.nix
+++ b/home-manager/modules/cli/tmux.nix
@@ -33,17 +33,24 @@
       bind -n C-j resize-pane -D 5
       bind -n C-k resize-pane -U 5
 
+      bind -r m resize-pane -Z
+
       # Custom keybinds
-      bind-key -T prefix g display-popup \
-      	-d "#{pane_current_path}" \
-      	-w 80% \
-      	-h 80% \
-      	-E "lazygit"
+      bind-key -T prefix f display-popup \
+      -d "#{pane_current_path}" \
+      -w 80% \
+      -h 80% \
 
       bind-key -T prefix o display-popup -E "tms"
 
       bind-key -T prefix n display-popup -E 'bash -i -c "read -p \"Session name: \" name; tmux new-session -d -s \$name && tmux switch-client -t \$name"'
 
+      bind-key -T prefix g display-popup \
+      -d "#{pane_current_path}" \
+      -w 80% \
+      -h 80% \
+      -E "lazygit"
+
       # Status Bar Configuration
       set-option -g status-position top  # Position status bar at the top
       set -g status-style bg=default
diff --git a/home-manager/modules/desktop/foot.nix b/home-manager/modules/desktop/foot.nix
index 5420f86..658383e 100644
--- a/home-manager/modules/desktop/foot.nix
+++ b/home-manager/modules/desktop/foot.nix
@@ -5,7 +5,7 @@
     settings = {
       main = {
         term = "xterm-256color";
-        font = "JetBrainsMono Nerd Font:size=12";
+        font = "JetBrains Mono Nerd Font:size=12, monospace:size=12";
         pad = "20x20";
         dpi-aware = "no";
       };
diff --git a/home-manager/modules/desktop/kitty.nix b/home-manager/modules/desktop/kitty.nix
index 7d94c7d..4a156f4 100644
--- a/home-manager/modules/desktop/kitty.nix
+++ b/home-manager/modules/desktop/kitty.nix
@@ -9,6 +9,7 @@
       size = 12;
     };
     settings = {
+      text_composition_strategy = "legacy";
       cursor_trail = 1;
       tab_bar_style = "powerline";
       tab_powerline_style = "round";
diff --git a/home-manager/modules/desktop/qtile/src/keybinds.py b/home-manager/modules/desktop/qtile/src/keybinds.py
index 85facf5..bcfc05d 100644
--- a/home-manager/modules/desktop/qtile/src/keybinds.py
+++ b/home-manager/modules/desktop/qtile/src/keybinds.py
@@ -16,32 +16,32 @@ def create_keys():
         EzKey("M-j", lazy.layout.down(), desc="Move focus down"),
         EzKey("M-k", lazy.layout.up(), desc="Move focus up"),
         EzKey("M-<space>", lazy.layout.next(), desc="Move window focus to other window"),
-        
+
         # Window movement
         EzKey("M-S-h", lazy.layout.shuffle_left(), desc="Move window to the left"),
         EzKey("M-S-l", lazy.layout.shuffle_right(), desc="Move window to the right"),
         EzKey("M-S-j", lazy.layout.shuffle_down(), desc="Move window down"),
         EzKey("M-S-k", lazy.layout.shuffle_up(), desc="Move window up"),
-        
+
         # Window resizing
         EzKey("M-C-j", lazy.layout.shrink(), desc="Shrink window"),
         EzKey("M-C-k", lazy.layout.grow(), desc="Grow window"),
         EzKey("M-n", lazy.layout.normalize(), desc="Reset all window sizes"),
-        
+
         # Layout controls
         EzKey("M-S-<Return>", lazy.layout.toggle_split(), desc="Toggle between split and unsplit sides of stack"),
         EzKey("M-z", lazy.next_layout(), desc="Toggle between layouts"),
-        
+
         # Window controls
         EzKey("M-S-q", lazy.window.kill(), desc="Kill focused window"),
         EzKey("M-f", lazy.window.toggle_fullscreen(), desc="Toggle fullscreen on the focused window"),
         EzKey("M-S-<space>", lazy.window.toggle_floating(), desc="Toggle floating on the focused window"),
-        
+
         # System controls
         EzKey("M-C-r", lazy.reload_config(), desc="Reload the config"),
         EzKey("M-C-q", lazy.shutdown(), desc="Shutdown Qtile"),
         EzKey("M-m", lazy.function(mode.toggle), desc="Toggle DND"),
-        
+
         # Applications
         EzKey("M-<Return>", lazy.spawn(APPS["terminal"]), desc="Launch terminal"),
         EzKey("M-d", lazy.spawn(APPS["launcher"]), desc="Exec app launcher"),
@@ -49,24 +49,24 @@ def create_keys():
         EzKey("M-b", lazy.spawn(APPS["browser"]), desc="Exec browser"),
         EzKey("M-c", lazy.spawn(APPS["editor"]), desc="Exec editor"),
         EzKey("M-<Tab>", lazy.spawn(APPS["notification_center"]), desc="Exec notification center"),
-        
+
         # Scripts
         EzKey("M-S-n", lazy.spawn("rofi-notes"), desc="Spawn rofi with notes script"),
         EzKey("M-C-t", lazy.spawn("toggle-theme"), desc="Exec theme switcher script"),
         EzKey("M-w", lazy.spawn("wallrandom"), desc="Exec random wallpaper script"),
         EzKey("M-C-p", lazy.spawn("wl-color-picker"), desc="Exec color picker application"),
-        
+
         # Scratchpad
         EzKey("M-s", lazy.group["scratchpad"].dropdown_toggle("Music")),
         EzKey("M-a", lazy.group["scratchpad"].dropdown_toggle("Term")),
-        
+
         # Screenshots
         EzKey("M-S-s", lazy.spawn("flameshot gui -c && wl-paste --type image/png > /tmp/clip.png", shell=True)),
         EzKey("M-o", lazy.spawn("eog /tmp/clip.png"), desc="Open last captured image"),
-        
+
         # Keyboard layout
         EzKey("S-<Tab>", lazy.widget["keyboardlayout"].next_keyboard()),
-        
+
         # Media keys - Audio
         EzKey("<XF86AudioRaiseVolume>", lazy.spawn("wpctl set-volume -l 1.4 @DEFAULT_AUDIO_SINK@ 5%+")),
         EzKey("<XF86AudioLowerVolume>", lazy.spawn("wpctl set-volume -l 1.4 @DEFAULT_AUDIO_SINK@ 5%-")),
@@ -74,18 +74,18 @@ def create_keys():
         EzKey("<XF86AudioPlay>", lazy.spawn("playerctl --player=spotify,%any play-pause")),
         EzKey("<XF86AudioPrev>", lazy.spawn("playerctl --player=spotify,%any previous")),
         EzKey("<XF86AudioNext>", lazy.spawn("playerctl --player=spotify,%any next")),
-        
+
         # Media keys - Brightness
         EzKey("<XF86MonBrightnessDown>", lazy.spawn("brightnessctl set 5%-")),
         EzKey("<XF86MonBrightnessUp>", lazy.spawn("brightnessctl set +5%")),
-        
+
         # MPD controls
         EzKey("M-p", lazy.spawn("mpc toggle"), desc="Pause/Unpause mpd player"),
         EzKey("M-S-<period>", lazy.spawn("mpc next"), desc="Mpd play next song"),
         EzKey("M-S-<comma>", lazy.spawn("mpc prev"), desc="Mpd play prev song"),
         EzKey("M-<semicolon>", lazy.spawn("mpc volume +5"), desc="Mpd volume up"),
         EzKey("M-S-<semicolon>", lazy.spawn("mpc volume -5"), desc="Mpd volume down"),
-        
+
         # KeyChord
         KeyChord([MOD], "i", [EzKey("M-i", lazy.ungrab_all_chords())], mode=True, name="Vm Mode"),
     ]
diff --git a/home-manager/modules/desktop/qtile/src/widgets.py b/home-manager/modules/desktop/qtile/src/widgets.py
index c40c6f1..917b013 100644
--- a/home-manager/modules/desktop/qtile/src/widgets.py
+++ b/home-manager/modules/desktop/qtile/src/widgets.py
@@ -25,7 +25,7 @@ def create_widget_list() -> list:
     from config import HOST
 
     base_color = colors["base15"] if colors["theme"] == "nord" else colors["base10"]
-    
+
     widgets = [
         widget.Image(
             filename=f"~/.config/qtile/assets/{colors["theme"]}-logo.png",
@@ -56,6 +56,7 @@ def create_widget_list() -> list:
         widget.Spacer(length=2),
         widget.CurrentLayout(
             foreground=colors["base09"],
+            mode="both",
             icon_first=True,
             custom_icon_paths=[f"~/.config/qtile/assets/layout/{colors["theme"]}"],
             scale=0.7,
@@ -121,7 +122,7 @@ def create_widget_list() -> list:
                 **powerline("forward_slash"),
             )
         )
-    
+
     widgets.extend([
         widget.TextBox(
             text="  ",
@@ -157,5 +158,5 @@ def create_widget_list() -> list:
             foreground=base_color,
         ),
     ])
-    
+
     return widgets
diff --git a/nixos/modules/programs.nix b/nixos/modules/programs.nix
index 1d51ce2..fb9a381 100644
--- a/nixos/modules/programs.nix
+++ b/nixos/modules/programs.nix
@@ -1,7 +1,7 @@
 {
   programs = {
     sway = {
-      enable = true;
+      enable = false;
       wrapperFeatures.gtk = true;
       extraPackages = [];
     };
